apps/web/pages/auth/login.tsx

```diff
@@ -57,7 +57,7 @@ inferSSRProps<typeof getServerSideProps> & WithNonceProps<{}>) {
         .string()
         .min(1, `${t("error_required_field")}`)
         .email(`${t("enter_valid_email")}`),
-      password: !!totpEmail ? z.literal("") : z.string().min(1, `${t("error_required_field")}`),
+      ...(!!totpEmail ? {} : { password: z.string().min(1, `${t("error_required_field")}`) }),
```

Comment:
The bug fixed in this PR https://github.com/calcom/cal.com/pull/10288 resurfaced again, so fixed that too in this PR